src/parse.test.ts

```diff
@@ -0,0 +1,104 @@
+import { parseGoList, parseGoModGraph } from './parse'
```

Comment:
Not going to lie, I kind of hate putting test files right alongside normal files ;) But I can let if fly...

Reply:
It's takes all all types! I think I've more strongly adopted this working on a TypeScript+Go project, where unit tests in Go *must* be collocated with the source files. So apologies for ry

Since this is to be open sourced, I'm inclined to keep the tests collocated because this is the common idiom for Jest test organization (it's the example given in the [Getting Started](https://jestjs.io/docs/getting-started) guide, for instance). But I fully bless internal TS projects using other organization mehtods.

I hope we can at least agree that that `__test__/` directory (another Jest practice) is an abomination ;)